examples/curb-detection/lifelong_learning_bench/testalgorithms/rfnet/unseen_sample_recognition_by_scene.py

```diff
@@ -0,0 +1,48 @@
+from typing import List, Any, Tuple
+
+from sedna.datasources import BaseDataSource
+from sedna.common.class_factory import ClassType, ClassFactory
+from sedna.algorithms.seen_task_learning.artifact import Task
+
+__all__ = ('UnseenSampleRecognitionByScene',)
+
+@ClassFactory.register(ClassType.UTD, alias="UnseenSampleRecognitionByScene")
+class UnseenSampleRecognitionByScene:
+    def __init__(self, **kwargs):
+        self.model_path = kwargs.get("model_path")
+
+    def __call__(self,
+                 samples: BaseDataSource, **kwargs):
+        from torch.utils.data import DataLoader
+        import torch
+        import torchvision
+        from torchvision import transforms
+        from PIL import Image
+        d_type = samples.data_type
+        x_data = samples.x
+        y_data = samples.y
+        data_transforms = transforms.Compose([
+                    transforms.Resize([32,32]),
+                    transforms.ToTensor()
+                    ])
+        #import pdb
+        #pdb.set_trace()
+        InferenceData=data_transforms(Image.open(x_data[0][0]))
+        device = 'cuda' if torch.cuda.is_available() else 'cpu'
+        pthfile = "./examples/curb-detection/lifelong_learning_bench/testalgorithms/rfnet/results/Epochofprose17.pth"
```

Comment:
The value of "pthfile" should be set in __init__ function by a passed parameter.